test/functional/page_objects/time_picker.js

```diff
@@ -90,6 +98,7 @@ export function TimePickerPageProvider({ getService, getPageObjects }) {
       await testSubjects.click('superDatePickerendDatePopoverButton');
       let panel = await this.getTimePickerPanel();
       await testSubjects.click('superDatePickerAbsoluteTab');
+      await testSubjects.click('superDatePickerAbsoluteDateInput');
```

Comment:
is it a general IE issue, that requires input click before typing or datePicker only?

Reply:
I'm not 100% sure this click is *really* needed for IE.  There could also be timing issues between clicking the absolute date button and starting to type in that field and this click happens to fix that (maybe just because it takes a few milliseconds to do the click)?   And or it could be that only on IE we have to get the focus into the field so that our native events typing of the input works.

In general, if it's a simple fix like this for IE, and it works across all our supported browsers, and doesn't take more than a second, I'd rather not conditionalize it based on browser.  

But on the other hand, I could see someone removing the click action while working on Chrome and having a hard time figuring out what broke IE.  So I guess I could be swayed either way on it.